### DISCUSSION TITLE
error resolveJsonModule' cannot be specified without 'node' of nextjs14 project .

### DISCUSSION THREAD

**Discussion by sytpb** [vercel/next.js#57740] [Help] [answered]
### Summary

`Option '--resolveJsonModule' cannot be specified without 'node' module resolution strategy.`

tsconfig.json
```
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "bundler",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ],
    "paths": {
      "@/*": ["./*"]
    }
  },
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
  "exclude": ["node_modules"]
}

```

### Additional information

```js
it is my first nextjs14 project .  created by npx create-next-app@latest .
```

### Example
![image](https://github.com/vercel/next.js/assets/12178686/d3925cd4-7e91-42e3-bfab-951bcdfda1b2)

_No response_

**Comment by icyJoseph** [comment]
Do you know what TS version you have? And if your Editor is using its own built-in TS version, or if its using the node_modules/package.json specified version?

If its VSCode, go to the tsconfig.json file and then do `ctrl/cmd + shift + p`.

Then type, `TypeScript: Select TypeScript Version`, and follow the instructions.

**Comment by sytpb** [comment]
![image](https://github.com/vercel/next.js/assets/12178686/f1425729-5b06-4137-8a5b-36c733a88061)

thanks for your reply , which one should I choose ?

**Reply by icyJoseph** [reply, marked as answer]
You should have the Workspace version. However, even that one is kind of low though. 

You should also update your VSCode, I think the default version today is 5.2.x, same as with next-app created projects, which is why I say that yours seems kind low.

Regardless, you should use the version in node_modules (workspace) because that's what the terminal and the deploy environments, will use when building your project.

**Reply by sytpb** [reply]
I have updated my VScode to latest with "typescript": "^5.2.2"   ,  now the error has gone .  I can run it up .  appreciate !
